### ISSUE TITLE
Request — Filter by relational field in the list view

### ISSUE THREAD

**Issue by Aurelsicoko** [strapi/rfcs#8]
This issue has been opened because we (as the core team) are looking for contributors to help us co-develop the feature.

[Link to the roadmap (related)](https://portal.productboard.com/strapi/1-public-roadmap/c/49-display-relational-field-in-list-view)

# Motivations
As we expect to display relational fields in the list view of the Content Manager, it now makes sense to filters the entries based on relational fields.

# Tasks

1. Find a contributor
2. Define the needs
3. Discuss a solution and technical implementation
4. Ask for design
5. Submit RFC

# Risks

- Don't forget to solve the issue whatever the kind of relationships. 
In other words, the list view must handle one-way, one-to-one, one-to-many, many-to-one, and many-to-many relationships.
- Make the filters too difficult to use and not readable.

**Comment by sam-pires** [comment]
Hello everyone, 

I posted a message in the issue #7 about Relational fields, I mentioned filtering by relation fields

**Comment by sam-pires** [comment]
https://github.com/strapi/rfcs/issues/7#issuecomment-740830097

**Comment by sam-pires** [comment]
Hello everyone, 
We released Strapi v3.4 last week. It includes the possibility to display Relational Fields in the List views, and to use them to sort and **filters**!

https://strapi.io/blog/product-updates-relational-fields-displayed-in-the-list-view

I'll close this issue.
Thanks for you involvement 👍
